<launch>
    <!-- 启动第一个相机节点（topic） -->
    <!-- <node name="xian_dj_visual_platform_mv_cs016_10gc_color_ros_topic_0" pkg="xian_dj_visual_platform_control_pkg" type="xian_dj_visual_platform_mv_cs016_10gc_color_ros_topic" output="screen">
        <param name="camera_index" value="0" />
        <param name="topic_name" value="xian_dj_hk_camera_0" />
    </node> -->

    <!-- 启动第一个相机节点（TCP server） -->
    <node name="xian_dj_visual_platform_mv_cs016_10gc_color_tcp_server_0" pkg="xian_dj_visual_platform_control_pkg" type="xian_dj_visual_platform_mv_cs016_10gc_color_tcp_server" output="screen">
        <param name="camera_index" value="0" />
    </node>


    
</launch>